app/stylesheets/boilerplate/language-selector.scss

```diff
@@ -1,10 +1,29 @@
+@import '../config/dimensions';
+@import '../config/colours';
 @import '../config/list';
+@import '../config/box';
+@import '../config/type';
 
-.language-selector {
+ul.language-selector {
   @include list-inline();
   @include list-unstyle();
+  @include border-radius(3px);
```

Comment:
should `3px` be in a variable? we don't have a lot of rounded corners, _maybe_ we can just leave this one hard-coded...